public/components/common/welcome/agents-welcome.js

```diff
@@ -321,21 +323,21 @@ class AgentsWelcome extends Component {
               <EuiFlexItem grow={false} style={{ marginTop: 7 }}>
                 <EuiButtonEmpty
                   iconType="inspect"
-                  onClick={() => this.props.switchTab('syscollector')}>
+                  onClick={() => this.props.switchTab('syscollector', notNeedStatus)}>
```

Comment:
**thought**: I was reviewing the source code and this could be affecting other logic. 

I guess setting the `force` parameter, you want to avoid this request: https://github.com/wazuh/wazuh-kibana-app/blob/7e173868b4c579cb391bf8f92fcfdc50529dec80/public/controllers/agent/agents.js#L503-L519

But this variable is used to set other logic, is this the expected behavior?
https://github.com/wazuh/wazuh-kibana-app/blob/7e173868b4c579cb391bf8f92fcfdc50529dec80/public/controllers/agent/agents.js#L547
https://github.com/wazuh/wazuh-kibana-app/blob/7e173868b4c579cb391bf8f92fcfdc50529dec80/public/controllers/agent/agents.js#L552
https://github.com/wazuh/wazuh-kibana-app/blob/7e173868b4c579cb391bf8f92fcfdc50529dec80/public/controllers/agent/agents.js#L556
https://github.com/wazuh/wazuh-kibana-app/blob/7e173868b4c579cb391bf8f92fcfdc50529dec80/public/controllers/agent/agents.js#L565 (by setting the `onlyAgent` and `preserveDiscover` variables)

Reply:
In those views it would not affect because it would not enter because they do not meet this [condition](https://github.com/wazuh/wazuh-kibana-app/blob/7e173868b4c579cb391bf8f92fcfdc50529dec80/public/controllers/agent/agents.js#LL564C6-L564C6).

```
      if (!this.ignoredTabs.includes(this.$scope.tab)) {
        this.$scope.switchSubtab(targetSubTab, true, onlyAgent, sameTab, preserveDiscover);
      }
```
Which is where these properties would be used